client/src/assets/images/global/copy.html

```diff
@@ -0,0 +1,6 @@
+<svg width="24px" height="24px" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
This is a [softies](https://robbiepearce.com/softies/) icon?

Reply:
No it's not, I will change it !